src/DDTrace/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -44,6 +44,19 @@ public function isArtisanQueueCommand()
             ]);
     }
 
+    public function getLoginFromArgs($args): string
+    {
+        $allowList = ["email", "username"];
+
+        foreach ($allowList as $key) {
+            if (isset($args[$key])) {
+                return $args[$key];
+            }
+        }
+
+        return null;
```

Comment:
why create a local variable and loop through it?... you can just write

```php
if (key_exists('email', $args)) {
  return $args['email'];
}
// ...
```

This would be more efficient and just plain easier to read

Reply:
Yeah that's right, I wanted to just have to modify the list if we needed to add / remove some fields but that's not really needed for only two fields